scripts/gen_dynamic_openapi.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Would it be better to modify the existing `gen_openapi.sh` rather than introducing a new one? If you do so, be sure to check whether it's run from CI and whether it still works after the change.

Reply:
Let me see if it's being used anywhere. I'm unclear on who actually uses the other script and I figured I'd leave it in case the structure of the registry file changes and it completely breaks this script.

Reply:
I think we want `gen_openapi.sh` script to be up to date with latest plugins and this change would ensure that.

Reply:
Sure! I just figured we were the only ones using it, so adding an additional script is nbd. But I can overwrite the old one.